docs/licenses.rst

```diff
@@ -10,12 +10,9 @@ Below are the open-source licenses for the SDK and its tools:
    licenses/sdk
    licenses/binarycsp
    licenses/cloud
-   licenses/greedy
    licenses/hybrid
-   licenses/neal
    licenses/dnx
    licenses/system
-   licenses/tabu
    licenses/minorminer
    licenses/penaltymodel
    licenses/preprocessing
```

Comment:
Needs a `licenses/samplers` line and consider for [dwave-samplers](https://github.com/dwavesystems/dwave-samplers/tree/main/docs) using an `sdk_index.rst` file for consistency (or removing in other repos, now we're not doing per-repo builds)